packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -515,6 +514,8 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
   // Map of index -> child state used manage where the dragging item will need
   // to be inserted.
   final Map<int, _ReorderableItemState> _items = <int, _ReorderableItemState>{};
+  // An eyeball value for auto scroll when the item was dragged to the edge.
+  static const double _kAutoScrollVelocityScaler = 100 / 14;
```

Comment:
Could this be something that lives in the `_EdgeDraggingAutoScroller` when it gets moved to a public class? Perhaps as the default value for the `velocityScaler` so that everything uses the same value for this by default?